packages/agent-toolkit/src/core/tools/platform-api-tools/all-monday-api-tool.ts

```diff
@@ -25,6 +25,13 @@ interface GraphQLResponse {
 export class AllMondayApiTool extends BaseMondayApiTool<typeof allMondayApiToolSchema> {
   name = 'all_monday_api';
   type = ToolType.ALL_API;
+  annotations = {
+    title: 'Run Query or Mutation on any monday.com API',
+    readOnlyHint: false,
+    destructiveHint: true,
+    idempotentHint: false,
+    openWorldHint: true,
+  };
```

Comment:
NIT -> this is prone to error, consider having three types of abstract classes:
1. BaseMondayApiToolSafe
2. BaseMondayApiToolDestructive
3. BaseMondayApiToolIdempotent

Or something along these lines

Reply:
Also `openWorldHint` is always true for our tools, so you can default it in `BaseMondayApiTool`